cloud/endagaweb/views/dashboard.py

```diff
@@ -560,6 +560,9 @@ def post(self, request, imsi=None):
                     CURRENCIES[currency]).amount_raw
             if abs(amount) > 2147483647:
                 raise ValueError(error_text)
+            if sub.balance + amount > network.max_account_limit:
+                error_text = 'Error : Crossed Credit Limit.'
```

Comment:
Print out the max limit in the error message? Also check your units. I am pretty sure `sub.balance` is actually in millicents/pesos depending on the currency selected. Testing this in USD is a good idea. Also can we write tests for this?